cryptkeeper.go

```diff
@@ -14,14 +14,17 @@ import (
 	"reflect"
 )
 
+/* NullString is an alias for sql.NullInt64 data type */
+type NullString sql.NullString
+
 /*CryptString is a wrapper for encrypting and decrypting a string for database operations. */
 type CryptString struct {
 	json.Marshaler
 	json.Unmarshaler
 	sql.Scanner
 	driver.Valuer
 
-	String string
+	String NullString
```

Comment:
it may be worth embedding the `NullString` struct rather than making a .String since that will cut out some code noise (ie `.String.String`, `.String.Valid`, etc)